### DISCUSSION TITLE
A way of seeing animation at a given point in time?

### DISCUSSION THREAD

**Discussion by chrisjallen** [pmndrs/react-spring#991] [Support]
Just reading through docs online, for crude example of my problem; if I was to imitate a timeline, say similar to the old days of flash, where I could scrub through say a few keyframes and the stage show the animation at that timeframe. Using the api, is there a deterministic way to render the animation at a given point in time? Am I missing some api Im unaware of?

Thanks 

Chris

**Comment by aleclarson** [comment]
Unfortunately not, because we don't use a spring algorithm that can generate keyframes. We might be open to making this possible in the future.

**Reply by dpnsoftdev** [reply]
@aleclarson Is it possible at lastest version?
